Tests/GitBuddyTests/Changelog/ChangelogCommandTests.swift

```diff
@@ -35,7 +35,7 @@ final class ChangelogCommandTests: XCTestCase {
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Tests/GitBuddyTests/Changelog/ChangelogCommandTests.swift;
  Line: 35;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)